pilot/pkg/proxy/envoy/config.go

```diff
@@ -78,6 +78,7 @@ func (conf *Config) Write(w io.Writer) error {
 
 // BuildConfig creates a proxy config with discovery services and admin port
 // it creates config for Ingress, Egress and Sidecar proxies
+// TODO: remove after new agent package is done
```

Comment:
I suggest to keep it and remove the hard-coded YAML instead.

Reply:
The hard coded yaml will be replaced with a template, this code is copied from old agent to 
keep it compiling, I'm working on removing most code here and use template only.